src/WatcherFactory.php

```diff
@@ -19,10 +19,12 @@ public static function create(array $options = []): array
         }
 
         $finder = (new Finder())
-            ->ignoreDotFiles(false)
-            ->ignoreVCS(false)
+            ->ignoreDotFiles($options['watch']['ignoreDotFiles'])
```

Comment:
Are you sure this won't blow up if `ignoreDotFiles` is not defined?

Reply:
I think it'll be guaranteed to be there once #116 is merged, but I'll verify that after addressing your feedback on that PR.

Reply:
Yeah, it's always there, now that #115 is merged.